.travis.yml

```diff
@@ -1,24 +1,10 @@
-language: objective-c
+language: node_js
 
 cache:
     directories:
         - node_modules
         - .nvm
 
-install:
-  - brew reinstall nvm
```

Comment:
Thanks! I had no idea what I was doing, I just copy and pasted that from react native